### PR TITLE
Shorten test sleep time

### PR DESCRIPTION
**Pull request by tcharding**
Shorten sleep times to 200 milliseconds instead of 1 second.  200 milliseconds is an eternity for a modern processor.  We do not need to wait for a second for the CPU to process things.  As for the CI infrastructure ...

### CODE REVIEW

**Comment by tcharding** [comment]
These patches were previously included in PR #1784.

File: api_tests/lib_sdk/cnd_instance.ts [hunk 1]
```diff
@@ -54,13 +57,15 @@ export class CndInstance {
...
-            );
+            if (global.verbose) {
+                console.log(
+                    `cnd ${this.actorConfig.name} exited with ${code ||
+                        "signal " + signal}`
+                );
+            }
         });
 
-        await sleep(1000); // allow the nodes to start up
```
Comment (bonomat) [inline api_tests/lib_sdk/cnd_instance.ts]: 📣Especially on my machine this won't be enough.

File: api_tests/lib_sdk/wallets/index.ts [hunk 2]
```diff
@@ -65,7 +65,7 @@ export async function pollUntilMinted(
     if (new BigNumber(currentBalance).gte(minimumBalance)) {
         return;
     } else {
-        await sleep(500);
```
Comment (bonomat) [inline api_tests/lib_sdk/wallets/index.ts]: 📣We have a new block every 1 second. Which means requesting the balance every 200ms does not make much difference besides putting more load on your machine.

File: api_tests/lib/actor.ts [hunk 3]
```diff
@@ -182,24 +185,28 @@ export class Actor {
...
+                            console.log(
+                                `Waiting for median time to pass %d`,
+                                min_median_block_time
+                            );
+                        }
 
                         let escapeSequence = "";
 
                         while (diff > 0) {
-                            await sleep(1000);
```
Comment (bonomat) [inline api_tests/lib/actor.ts]: 📣Our HTLC timeouts are set to +3/+8 seconds, so asking the chain every 200ms for the latest time won't make much difference.

**Comment by tcharding** [comment]
I put the verbose flag patch into a separate PR (merged already) and forgot to remove it from this one.  Closing this since after the comments, thanks for the review @bonomat

**Pull request closed without merging**